### DISCUSSION TITLE
Group .psd layers using ImageMagick

### DISCUSSION THREAD

**Discussion by aamirencodes** [ImageMagick/ImageMagick#4664] [Help] [answered]
Hi Community,

I'm trying to generate an .psd file using the below command and it seems to be working as expected. In the layers section of photoshop I'm able to view the images with the layer names with which I have specified with the "-label" attribute in the command. One of the things that I'm trying to achieve is create a group of input layers (kind of hierarchy where a set of images appear under a group when final psd artwork is viewed via photoshop). Can you please help me with your expertise?

`magick test_image1.png -label test_image2 test_image2.jpg -label test_image3 test_image3.png -label test_image1 test_image1.png -label final_artwork.psd`

Thanks

**Comment by aamirencodes** [comment]
@snibgo @fmw42 Can you please help us with your expertise here?

**Reply by dlemstra** [reply]
Please don't notify people by name. Someone will respond when they have time.

**Reply by aamirencodes** [reply]
Apologies, will keep a note of that. Thanks

**Comment by fmw42** [comment]
I do not believe that ImageMagick supports layer groups in creating new photoshop files. However, I will defer to the ImageMagick developers.

**Reply by aamirencodes** [reply]
Thank you for the response.

**Comment by snibgo** [comment, marked as answer]
I don't use Photoshop. IM has no mechanism for grouping images into a hierarchy.

**Reply by aamirencodes** [reply]
Thanks for your response.
